### PR TITLE
Added script to scrape PDF #29

### PR DESCRIPTION
**Pull request by kritikaparmar-programmer**
# Description:
Added script to scrape PDF

# Issue- #29

### CODE REVIEW

**Review by tejan-singh — CHANGES_REQUESTED** [review]
please try to add pdf file as input thourgh command line.

File: ScrapePDF/Readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Script to scrape pdf
+
+## Overview:
+- A beginner friendly script to scrape pdf. You can easily get document info sunch as creator , crceation_date and no. of pages. Extract as many pages as you want.
```
Comment (tejan-singh) [inline ScrapePDF/Readme.md]: please fix spelling mistakes in readme.

**Comment by kritikaparmar-programmer** [comment]
Hey @tejan-singh kindly take a look at the changes.

**Review by tejan-singh — CHANGES_REQUESTED** [review]
You have not included, how to use this script? in the readme. It would be better to explain a use-case.

**Comment by kritikaparmar-programmer** [comment]
Done @tejan-singh

**Pull request merged**